<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="3" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-12-21T13:44:55 GMT" name="Default suite" finished-at="2023-12-21T13:45:11 GMT" duration-ms="15419">
    <groups>
    </groups>
    <test started-at="2023-12-21T13:44:55 GMT" name="Default test" finished-at="2023-12-21T13:45:11 GMT" duration-ms="15419">
      <class name="end_2_end.Result_Page_e2e">
        <test-method is-config="true" signature="setup()[pri:0, instance:end_2_end.Result_Page_e2e@3059cbc]" started-at="2023-12-21T13:44:55 GMT" name="setup" finished-at="2023-12-21T13:45:01 GMT" duration-ms="5154" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="CheckFor5Beds_Testcase_003()[pri:0, instance:end_2_end.Result_Page_e2e@3059cbc]" started-at="2023-12-21T13:45:01 GMT" name="CheckFor5Beds_Testcase_003" finished-at="2023-12-21T13:45:09 GMT" duration-ms="7988" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at pages.Result_Page.verifyResults(Result_Page.java:285)
at end_2_end.Result_Page_e2e.CheckFor5Beds_Testcase_003(Result_Page_e2e.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckFor5Beds_Testcase_003 -->
        <test-method is-config="true" signature="teardown(org.testng.ITestResult)[pri:0, instance:end_2_end.Result_Page_e2e@3059cbc]" started-at="2023-12-21T13:45:09 GMT" name="teardown" finished-at="2023-12-21T13:45:11 GMT" duration-ms="2222" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=CheckFor5Beds_Testcase_003 status=FAILURE method=Result_Page_e2e.CheckFor5Beds_Testcase_003()[pri:0, instance:end_2_end.Result_Page_e2e@3059cbc] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- end_2_end.Result_Page_e2e -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
